google_calendar/google-calendar.lua

```diff
@@ -0,0 +1,119 @@
+--[[
+
+# [Google Calendar API](https://developers.google.com/google-apps/calendar/v3/reference/) connector.
+
+### Authorization
+
+Authorization use [Basic authentication](console.developers.google.com).
```

Comment:
you implemented Oauth authentication, but referred to Basic authentication and for some reason not to documentation, but to developer console.